src/modules/Dashboard/components/HeaderOptions/HeaderOptions.tsx

```diff
@@ -0,0 +1,50 @@
+import { Box, Button } from '@mui/material';
+import { useState } from 'react';
+import { Link, generatePath, useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+
+import { page } from 'resources';
+import { Svg } from 'shared/components';
+import { ExportDataSetting } from 'shared/features/AppletSettings';
+import { StyledTitleMedium, theme, variables } from 'shared/styles';
+import { Mixpanel } from 'shared/utils';
+
+export const HeaderOptions = () => {
+  const [isExportOpen, setIsExportOpen] = useState(false);
+  const { t } = useTranslation('app');
+  const { appletId } = useParams();
+  const isSettingsSelected = location.pathname.includes('settings');
+
+  return (
+    <Box sx={{ display: 'flex', gap: theme.spacing(1), placeItems: 'baseline' }}>
```

Comment:
In cases where the ``sx`` prop contains more than three properties, it would be beneficial to consider using styled components. This approach can enhance the readability and maintainability of the code. Please check other occurrences as well.